contracts/misc/ProtocolDataProvider.sol

```diff
@@ -15,6 +15,7 @@ import {INToken} from "../interfaces/INToken.sol";
 import {IPToken} from "../interfaces/IPToken.sol";
 import {IPool} from "../interfaces/IPool.sol";
 import {IProtocolDataProvider} from "../interfaces/IProtocolDataProvider.sol";
+import "hardhat/console.sol";
```

Comment:
should be removed

Reply:
sorry, totally missed that, thanks. Restored.